### PR TITLE
Fix for seg fault when a xray tile has no data refs #51

### PR DESCRIPTION
**Pull request by timwaters**
Patch to fix a seg fault seen in the data inspector layer if there is no data in the tile.

### CODE REVIEW

File: src/back/XRayTile.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ var XRayTile = function (z, x, y, data, options) {
 XRayTile.prototype.render = function (project, map, cb) {
     var styleMap = this.styleMap(project),
         vtile = new mapnik.VectorTile(this.z, this.x, this.y);
-    vtile.setData(this.data);
+    if (this.data.length > 0) vtile.setData(this.data);
```
Comment (yohanboniface) [inline src/back/XRayTile.js]: `if (this.data.length)` should be enough :)

**Comment by timwaters** [comment]
thanks, added

**Comment by timwaters** [comment]
can rebase / squah into one commit if that helps :)

**Comment by yohanboniface** [comment]
Excellent, thanks! :)

**Pull request merged**